File: programming/labaratory-work-1/solutions/solution-1/main.c
#include <stdio.h>

int main() {
  int n = 0;

  scanf_s("%d", &n);

  int prev_k = 0;
  int count = 0;
  int max_count = 0;

  for (int i = 0; i < n; i++) {
    int k = 0;

    scanf_s("%d", &k);

    if (k > prev_k) {
      count++;
    }

    if (k <= prev_k) {
      count = 1;
    }

    if (count > max_count) {
      max_count = count;
    }
  }

  printf_s("%d", max_count);

  return 0;
}

File: programming/labaratory-work-1/solutions/solution-1/go/main.go
package main

import "fmt"

func main() {
	var n int

	fmt.Scanf("%d", &n)

	var prevK int = 0
	var count int = 0
	var maxCount int = 0

	for i := 0; i < n; i++ {
		var k int

		fmt.Scanf("%d", &k)

		if k > prevK {
			count++
		}

		if k <= prevK {
			count = 1
		}

		if (count > maxCount) {
			maxCount = count
		}
	}

	fmt.Printf("%d", maxCount)
}